src/__tests__/Utils-spec.js

```diff
@@ -13,9 +13,11 @@ import {
   mapNativeEventNames,
 } from '../Utils';
 import {
-  describeWithDOM,
   mount,
 } from '../';
+import {
+	describeWithDOM,
```

Comment:
theoretically, but i haven't gotten around to these rules yet, so i don't know how they're configured :-)